Composer/packages/extensions/visual-designer/src/widgets/ListOverviewWidget.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { generateSDKTitle } from '@bfc/shared';
+import React from 'react';
+
+import { ObiColors } from '../constants/ElementColors';
+import { ListOverviewCard } from '../components/nodes/templates/ListOverviewCard';
+import { ListOverview } from '../components/common/ListOverview';
+import { WidgetContainerProps } from '../schema/uischema.types';
+import { NodeMenu } from '../components/menus/NodeMenu';
+
+export interface ListOverviewWidgetProps extends WidgetContainerProps {
+  title: string;
+  content: { [key: string]: any };
+  menu?: JSX.Element | string;
+  children?: JSX.Element;
+  size?: {
+    width: number;
+
+    height: number;
+  };
+  itemSize?: {
+    width: number;
+    height: number;
+    marginTop: number;
+  };
+  colors?: {
+    theme: string;
+    icon: string;
+  };
+}
+
+const DefaultCardColor = {
+  theme: ObiColors.AzureGray3,
+  icon: ObiColors.AzureGray2,
+};
+
+export const ListOverviewWidget = Component => {
+  return class extends React.Component<ListOverviewWidgetProps, object> {
+    renderItems(items, itemSize, Render) {
+      if (!Array.isArray(items)) {
+        return null;
+      }
+      const ItemRender = ({ children }) => (
+        <Render
+          css={{
+            height: itemSize.height,
+            width: itemSize.width,
+            marginTop: itemSize.marginTop,
+          }}
+        >
+          {children}
+        </Render>
+      );
+      return (
+        <div css={{ padding: '0 0 8px 8px' }}>
+          <ListOverview
+            items={items}
+            ItemRender={ItemRender}
+            maxCount={3}
+            styles={{
+              height: itemSize.height,
+              width: itemSize.width,
+              marginTop: itemSize.marginTop,
+            }}
```

Comment:
item size should be part of ItemRenderer, in your case, should be binded to the styled component you passed in as `Component`.

Reply:
The problem here is you passed in a pair of `weight` and `height`, intuitively, it should limit the whole list's size but actually it's for a single item.